packages/core/src/browser/tree/tree.ts

```diff
@@ -206,7 +221,7 @@ export class TreeImpl implements Tree {
     protected readonly toDispose = new DisposableCollection();
 
     protected nodes: {
-        [id: string]: TreeNode | undefined
+        [id: string]: (TreeNode & MutableTreeNode) | undefined
```

Comment:
`Mutable<TreeNode>`

Reply:
Thanks for pointing out the Mutable type.  I've switched to use it.